servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1081,6 +1081,9 @@ void RenderForwardClustered::_fill_render_list(RenderListType p_render_list, con
 					distance = -distance_max;
 				}
 
+				if (p_render_data->cam_ortogonal)
+					distance = 1.0;
```

Comment:
Braces are required around one-line `if` statements:

```suggestion
				if (p_render_data->cam_ortogonal) {
					distance = 1.0;
				}
```

(Note for other reviewers: `cam_ortogonal` is the correct spelling for the variable; we should probably fix the typo in a separate PR.)

Reply:
Thanks for linking the guidelines, will take into account in the future, first time contributing